lib/galaxy/managers/libraries.py

```diff
@@ -208,6 +210,8 @@ def list(self, trans, deleted: Optional[bool] = False) -> Tuple[Query, Dict[str,
                 prefetched_ids["allowed_library_add_ids"] = allowed_library_add_ids
                 prefetched_ids["allowed_library_modify_ids"] = allowed_library_modify_ids
                 prefetched_ids["allowed_library_manage_ids"] = allowed_library_manage_ids
+        sort_column = getattr(Library, "name")
+        query = query.order_by(asc(func.lower(sort_column)))
```

Comment:
why not just `Library.name` instead of `sort_column`?

Reply:
I did that in a subsequent commit when linter yelled at me. I used to have something else there but there is actually no more options for this endpoint.